draft-ietf-mls-protocol.md

```diff
@@ -3547,7 +3550,8 @@ message at the same time, by taking the following steps:
   field populated from Proposals received during the current epoch, and an empty
   `path` field.
 
-* Generate the provisional ratchet tree and GroupContext by applying the proposals
+* Generate the provisional ratchet tree and GroupContext by applying the valid,
+  compatible proposals
```

Comment:
```suggestion
* Generate the provisional ratchet tree and GroupContext by applying the proposals
```

Reverting because the choice of proposals happens earlier.  All proposals included in the initial commit need to be applied.